### ISSUE TITLE
dependency does not exist

### ISSUE THREAD

**Issue by lauralarez** [BlinkID/blinkid-flutter#2]
Hi, I created a new project and I am trying to add the dependency on my pubspec.yaml file as stablished on README:

```
dependencies:
  ...
  blinkid_flutter:
```

However it throws the following exception:

> "Because project_name depends on blinkid_flutter any which doesn't exist (could not find package blinkid_flutter at https://pub.dartlang.org), version solving failed."

When I looked it up in https://pub.dartlang.org it really does not exist.

**Comment by SandraKuzmic** [comment]
Hi, 

blinkid_flutter is not yet published since we are still implementing the android part. In order to use the current version of blinkid_flutter You need to set the path to cloned repository plugin:

```
dependencies:
  blinkid_flutter:
    path: path/to/BlinkID
```

You can also use `initFlutterDemo.sh` script to generate the sample and check its code for more info.

In the meantime, we will update the readme so it is not confusing.

Best regards,
Sandra

**Comment by SandraKuzmic** [comment]
Hi,

just wanted to keep You updated, blinkid_flutter is now published and can be found [here](https://pub.dev/packages/blinkid_flutter).

Best regards,
Sandra

**Comment by ArtemKravchenko** [comment]
If I add BlinkID as described in the README file it will turn down all the other Android plugins :(

**Comment by SandraKuzmic** [comment]
Hi, 

can you please share your pubspec.yaml file?

Here is a part of my test pubspec.yaml and all dependencies worked fine when used in main.dart.
```
dependencies:
  flutter:
    sdk: flutter
    
  blinkid_flutter:
  battery: ^1.0.1
  cupertino_icons: ^0.1.2
``` 

Best regards,
Sandra

**Comment by ArtemKravchenko** [comment]
dependencies:
  flutter:
    sdk: flutter

  blinkid_flutter:
  url_launcher: ^5.4.7

**Comment by SandraKuzmic** [comment]
Hi @ArtemKravchenko, 

thank you for pointing out the issue, we are working on the fix.

For now, can you use provided MainActivity and just add desired plugin manually? Something like this:
```
import android.os.Bundle;
import io.flutter.app.FlutterActivity;
import com.microblink.flutter.MicroblinkFlutterPlugin;
import io.flutter.plugins.urllauncher.UrlLauncherPlugin;

public class MainActivity extends FlutterActivity {
  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

    MicroblinkFlutterPlugin.registerWith(registrarFor("com.microblink.flutter.MicroblinkFlutterPlugin"));
    UrlLauncherPlugin.registerWith(registrarFor("io.flutter.plugins.urllauncher.UrlLauncherPlugin"));
  }
}
```

Best regards,
Sandra

**Comment by ArtemKravchenko** [comment]
We didn't edit MainActivity file in this way. Thank you for advice.

**Comment by SandraKuzmic** [comment]
Hi @ArtemKravchenko 

we added support for FlutterEngine and it is now the default way of registering BlinkID plugin.

The fix is released as part of the [v5.7.0 release](https://github.com/BlinkID/blinkid-flutter/releases/tag/v5.7.0).

Best regards,
Sandra

**Comment by anjapenic** [comment]
Hello all,

Since this was fixed in v5.7.0, we're closing the issue. If you need any further assistance, feel free to contact us at support@microblink.com or open a new issue.

Best regards,
Anja
